indigo_social/default_badges.py

```diff
@@ -24,35 +24,53 @@ class ContributorBadge(PermissionBadge):
 
 class DrafterBadge(PermissionBadge):
```

Comment:
Do things go wrong if we change this class name, and just retain the `slug` for backwards compatibility?